huggle/Localization/en.xml

```diff
@@ -159,7 +159,7 @@
   <string name="config-html-messages">HAN can render HTML messages (dangerous)</string>
   <string name="config-summary-present">Highlight summary when present instead of when it&apos;s missing</string>
   <string name="config-close-without">Close without saving</string>
-  <string name="config-confirmmultiple">Confirm multiple reversions of an edit by the same user</string>
```

Comment:
this probably should be "revert of multiple revisions"

Reply:
Alright, thanks for the feedback. Edited it and committed.